alerts/kube_apiserver.libsonnet

```diff
@@ -3,8 +3,10 @@ local utils = import 'utils.libsonnet';
 {
   _config+:: {
     kubeApiserverSelector: error 'must provide selector for kube-apiserver',
+    kubeApiserverRequestTerminationsSelector: error 'must provide selector for kube-apiserver-request-terminations',
 
     kubeAPILatencyWarningSeconds: 1,
+    kubeAPIRequestTerminationsPercent: 0.20,
```

Comment:
sorry tiny thing, we try to keep these not configurable until there is a real need for it

Reply:
:+1: fixed! good catch on the selector.